docs/developer/Code_of_Conduct.md

```diff
@@ -0,0 +1,38 @@
+# OADP Team Code of Conduct
+
+## Introduction
+
+The guidelines here should be clear, easy to understand, and easy to follow.
+
+### Assume positive intent, and bring a positive attitude and intentions.
+
+First and foremost, assume positive intent. We are all working towards the same goal, and we are all part of the same team. We should bring a positive attitude and intentions to our work.  This work can be fun and challenging, and we should enjoy the journey.
+
+### Respect peoples time and effort.
+
+If someone takes the time to review your PR or contributes to your issue please understand
+those comments took time and effort. Respect that time and effort and respond to it with
+respect.
+
+
+### Pull Request guidelines
+1. Follow the PR template instructions. 
+1. Avoid self overrides. If prow tests are failing, ask the reviewers to override tests if needed. 
```

Comment:
Do we also want to address when overrides are appropriate, at least in part? Something like "Always allow a required test to run and fail first before overriding" if we want to make a hard rule there. Otherwise, "Never override unless we expect the test to fail consistently."

Reply:
I think we should have the discussion first.  Not sure if I'm right, but IMHO I said someone that is currently on the CI team should look at the failure and override.  This way we can catch the repeat issues and appropriately add to @mpryc 's retry mechanism.  Open for discussion and clarifying

Reply:
can we enforce who can override through OWNERS file?

Reply:
@mateusoliveira43 The enforcement we're talking about here isn't for users who shouldn't be in OWNERS, though. i.e. if someone who is an owner submits a PR, that person shouldn't override. I guess the real question is whether the prow bot can enforce rules like "don't allow /override from author".